app/models/agents/commander_agent.rb

```diff
@@ -17,7 +17,7 @@ class CommanderAgent < Agent
 
       * `enable`: Target Agents are enabled (if not) when this agent is triggered.
 
-      * `configure`: Target Agents have their options updated with the contents of `configure_options`.
+      * `configure`: Target Agents have their options updated with the contents of `configure_options`. Set `configure_options_nested_array_update_mode` to `merge`(default) or `replace` when configuring arrays that are nested within the `configure_options` hash.
```

Comment:
I think the default should be `replace` because that's what we do now, so if anyone is depending on that behavior, we don't want to break it for them.